charts/sriov/100.0.1+up0.1.0/Chart.yaml

```diff
@@ -17,7 +17,7 @@ icon: https://charts.rancher.io/assets/logos/sr-iov.svg
 keywords:
 - sriov
 - Networking
-kubeVersion: '>= 1.16.0-0'
+kubeVersion: '>= 1.16.0-0 < 1.22.0-0'
```

Comment:
should not be lowering scope of kube-version, changing the annotation is sufficient enough.